.travis.yml

```diff
@@ -52,13 +39,16 @@ matrix:
     - name: "linux.x64.release"
       addons:
         apt:
-          packages: [ autoconf, automake, libtool, ninja-build, libicu-dev ]
+          packages: [ autoconf, automake, libtool, ninja-build, libicu-dev, npm ]
+      install:
+        - npm install
       script:
-        - cmake -H. -Bout/linux/x64/release -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=release -DESCARGOT_OUTPUT=bin -GNinja
+        - cmake -H. -Bout/linux/x64/release -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=release -DESCARGOT_OUTPUT=bin -DVENDORTEST=1 -GNinja
         - ninja -Cout/linux/x64/release
         - cp ./out/linux/x64/release/escargot ./escargot
         - travis_wait tools/run-tests.py --arch=x86_64 jetstream-only-simple
         - tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js test262 internal octane chakracore jsc-stress regression-tests
+        - tools/run-tests.py --arch=x86_64 v8 spidermonkey
```

Comment:
These could be merged into the line above.

Reply:
This line is merged as you commented. Thank you for your review.